packages/application-generic/src/factories/mail/handlers/mailgun.handler.ts

```diff
@@ -15,10 +15,10 @@ export class MailgunHandler extends BaseHandler {
       domain: string;
       from: string;
     } = {
-      apiKey: credentials.apiKey,
-      username: credentials.user,
-      domain: credentials.domain,
-      from,
+      apiKey: credentials.apiKey as string,
+      username: credentials.user as string,
+      domain: credentials.domain as string,
+      from: from as string,
```

Comment:
Seems like we have a duplication of all the provider factories on both API and application generic. didn't refactored it for now to avoid non strict null changes in this PR.